picard/ui/mainwindow.py

```diff
@@ -737,14 +742,30 @@ def _update_cd_lookup_button(self):
     def toggle_rename_files(self, checked):
         config = get_config()
         config.setting["rename_files"] = checked
+        self.update_settings_in_profile()
 
     def toggle_move_files(self, checked):
         config = get_config()
         config.setting["move_files"] = checked
+        self.update_settings_in_profile()
```

Comment:
I'm not fond of this mechanism. Does this mean it has to be called everytime the value is changed to be stored in profile?
Actually not even sure to understand this part.